### PR TITLE
Added eliza client methods to mess with participants

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim I'm doing my best to maintain internal APIs while you make them more flexible

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/ElizaDiscussionCommander.scala [hunk 1]
```diff
@@ -138,6 +139,16 @@ class ElizaDiscussionCommanderImpl @Inject() (
       }
     }
   }
+  def changeKeepParticipants(userId: Id[User], keepId: Id[Keep], addUsers: Set[Id[User]], removeUsers: Set[Id[User]])(implicit context: HeimdalContext): Future[Unit] = {
+    for {
+      thread <- getOrCreateMessageThreadForKeep(keepId)
+      _ = require(thread.allParticipants.contains(userId))
+      _ = require(removeUsers.isEmpty) // TODO(ryan): really need to revisit this
+      // TODO(ryan): This is horrible. We pass in Id[User], immediately go BACK to shoebox to convert to ExternalId[User], THEN TURN AROUND AGAIN TO CONVERT BACK TO Id[User]
```
Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/ElizaDiscussionCommander.scala:147]: @leogrim this is why I prefer `PublicId`

Comment (camhashemi) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/ElizaDiscussionCommander.scala:147]: +1. it's nice when you don't have to do this.

**Comment by camhashemi** [comment]
after you merge, we should add this to grant `KeepPermission.VIEW_KEEP` and `KeepPermission.ADD_COMMENTS`

**Comment by ryanpbrewster** [comment]
:+1:. Rather, we should only call this method once we have added those permissions.

**Pull request closed without merging**